$NetBSD: patch-aa,v 1.1.1.1 2004/02/24 21:47:25 jmmv Exp $

Do not make the program depend on -ljpeg (GNU make looks for it in the build machine's library directories).

--- Makefile.orig
+++ Makefile
@@ -36,9 +36,9 @@
 ##############
 # JPEG STUFF #
 ##############
-JPEG_LIB = jpeg/libjpeg.a
-JPEG_DIR = -Ijpeg
-JPEG_LIB_DIR = -Ljpeg
+JPEG_LIB = -ljpeg
+JPEG_DIR = 
+JPEG_LIB_DIR = 
 MP_JPEG_OBJS = jpeg.o
 MP_JPEG_SRCS = jpeg.c
 
@@ -57,7 +57,7 @@
 #################
 # INCLUDE FILES # 
 #################
-INCLUDEDIR = -Iheaders -I/usr/include $(JPEG_DIR)
+INCLUDEDIR = -Iheaders $(JPEG_DIR)
 
 ############################################################################
 # LIBRARIES # specify library directories; need jpg lib and maybe pbm	   #
@@ -202,8 +202,8 @@
 # BINARIES #
 ############
 
-mpeg_encode: $(MP_ALL_OBJS) $(JPEG_LIB)
-	$(PURIFY) $(CC) $(CFLAGS) -o $@ $(MP_ALL_OBJS) $(LIBDIRS) $(LIBS)
+mpeg_encode: $(MP_ALL_OBJS)
+	$(PURIFY) $(CC) $(LDFLAGS) -o $@ $(MP_ALL_OBJS) $(LIBDIRS) $(LIBS)
 	strip mpeg_encode
 
 profile: $(MP_ALL_OBJS)
